usecases/objects/update.go

```diff
@@ -15,6 +15,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/weaviate/weaviate/usecases/memwatch"
+
```

Comment:
Can you sort all these new imports? In each case you just need to remove the newline between the newly imported pkg, and the rest of the non-stdlib imports